### PR TITLE
Update QSG steps

### PR DESCRIPTION
**Pull request by binodmx**
Update Quick Start Guide steps to run APK properly with latest changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,77 +89,92 @@ WSO2 API Kubernetes Platform has released following docker images in the WSO2 pu
...
+    >
+    > Provide the router service external ip to `{router_service}` in below commands.
+
+3. Get a token to invoke the System API.
+    ```bash
+    ACCESS_TOKEN=$(curl --location --request POST '{router_service}:9095/oauth2/token' \
+    --header 'Host: idp.am.wso2.com' \
+    --header 'Authorization: Basic NDVmMWM1YzgtYTkyZS0xMWVkLWFmYTEtMDI0MmFjMTIwMDAyOjRmYmQ2MmVjLWE5MmUtMTFlZC1hZmExLTAyNDJhYzEyMDAwMg==' \
+    --header 'Content-Type: application/x-www-form-urlencoded' \
+    --data-urlencode 'grant_type=client_credentials' | jq -r '.access_token')
```
Comment (AmaliMatharaarachchi) [inline README.md:153]: this command implies, that it requires the jq, isn't it?

File: README.md [hunk 2]
```diff
@@ -89,77 +89,92 @@ WSO2 API Kubernetes Platform has released following docker images in the WSO2 pu
...
+    >
+    > Services should be created in a different namespace than APK or Kubernetes System namespaces.
+    >
+    > APIs should be created in the APK deployment namespace.
+    >
+    > Provide the router service external ip to `{router_service}` in below commands.
+
+3. Get a token to invoke the System API.
+    ```bash
+    ACCESS_TOKEN=$(curl --location --request POST '{router_service}:9095/oauth2/token' \
```
Comment (AmaliMatharaarachchi) [inline README.md:149]:
```suggestion
    ACCESS_TOKEN=$(curl --location --request POST 'https://{router_service}:9095/oauth2/token' ### \
```

File: README.md [hunk 3]
```diff
@@ -89,77 +89,92 @@ WSO2 API Kubernetes Platform has released following docker images in the WSO2 pu
...
+    --header 'Host: idp.am.wso2.com' \
+    --header 'Authorization: Basic NDVmMWM1YzgtYTkyZS0xMWVkLWFmYTEtMDI0MmFjMTIwMDAyOjRmYmQ2MmVjLWE5MmUtMTFlZC1hZmExLTAyNDJhYzEyMDAwMg==' \
+    --header 'Content-Type: application/x-www-form-urlencoded' \
+    --data-urlencode 'grant_type=client_credentials' | jq -r '.access_token')
+    ```
+
+4. List the created API and retrieve API's `id`.
+    ```bash
+    curl --location --request GET '{router_service}:9095/api/am/runtime/apis' \
```
Comment (AmaliMatharaarachchi) [inline README.md:158]:
```suggestion
    curl --location --request GET 'https://{router_service}:9095/api/am/runtime/apis' \
```

File: README.md [hunk 4]
```diff
@@ -89,77 +89,92 @@ WSO2 API Kubernetes Platform has released following docker images in the WSO2 pu
...
---header 'Authorization: Bearer $TOKEN')
-  ```
 6. Invoke the API.
-  ```bash
-  curl --location --request GET '{router_service}:9095/http-bin-api/1.0.8/get' \
---header 'HOST: gw.wso2.com' \
---header 'Internal-Key: $INTERNAL_KEY'
-  ```
+    ```bash
+    curl --location --request GET '{router_service}:9095/http-bin-api/1.0.8/get' \
```
Comment (AmaliMatharaarachchi) [inline README.md:174]:
```suggestion
    curl --location --request GET 'https://{router_service}:9095/http-bin-api/1.0.8/get' \
```

Comment (binodmx) [reply README.md]: Added by https://github.com/wso2/apk/pull/1104/commits/8a03a2456903cff6261b62e40e9853326e95cf5a

File: README.md (hunk 3 above)
Comment (binodmx) [reply README.md]: Added by https://github.com/wso2/apk/pull/1104/commits/8a03a2456903cff6261b62e40e9853326e95cf5a

File: README.md [hunk 5]
```diff
@@ -89,77 +89,92 @@ WSO2 API Kubernetes Platform has released following docker images in the WSO2 pu
...
+4. List the created API and retrieve API's `id`.
+    ```bash
+    curl --location --request GET '{router_service}:9095/api/am/runtime/apis' \
+    --header 'Host: api.am.wso2.com' \
+    --header 'Authorization: Bearer $ACCESS_TOKEN'
+    ```
+
+5. Get a token to invoke the created API. Provide the API's `id` to `{api_id}` in below command.
+    ```bash
+    INTERNAL_KEY=$(curl --location --request POST '{router_service}:9095/api/am/runtime/apis/{api_id}/generate-key' \
```
[the remark this answers is not in the document]
Comment (binodmx) [reply README.md]: Added by https://github.com/wso2/apk/pull/1104/commits/8a03a2456903cff6261b62e40e9853326e95cf5a

File: README.md (hunk 2 above)
Comment (binodmx) [reply README.md]: Added by https://github.com/wso2/apk/pull/1104/commits/8a03a2456903cff6261b62e40e9853326e95cf5a

File: README.md (hunk 1 above)
Comment (binodmx) [reply README.md:153]: yes

File: README.md [hunk 6]
```diff
@@ -89,77 +89,92 @@ WSO2 API Kubernetes Platform has released following docker images in the WSO2 pu
 ### Before you begin...
 
 * Install [Helm](https://helm.sh/docs/intro/install/)
```
Comment (RakhithaRR) [inline README.md:91]: Shall we add the helm version as 3.11.x?

Comment (binodmx) [reply README.md]: Added by https://github.com/wso2/apk/pull/1104/commits/a6d95d94705d687490dfd85ed65053bb2a0c6473

**Review by RakhithaRR — APPROVED** [review]

**Pull request merged**